### PR TITLE
Find&Replace coreclr/src -> coreclr

### PR DESCRIPTION
**Pull request by jkotas**

### CODE REVIEW

File: src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs [hunk 1]
```diff
@@ -2491,7 +2491,7 @@ private RuntimeType[] GetNestedTypes_internal(string? displayName, BindingFlags
             get
             {
                 // See https://github.com/mono/mono/issues/18180 and
-                // https://github.com/dotnet/runtime/blob/f23e2796ab5f6fea71c9fdacac024822280253db/src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs#L1468-L1472
+                // https://github.com/dotnet/runtime/blob/f23e2796ab5f6fea71c9fdacac024822280253db/src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs#L1468-L1472
```
Comment (jkotas) [inline src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs:2494]:
```suggestion
                // https://github.com/dotnet/runtime/blob/69e114c1abf91241a0eeecf1ecceab4711b8aa62/src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs#L1505-L1509
```

**Review by ViktorHofer — APPROVED** [review]
Speed reviewed. Thanks. Let's try to get this in asap.

**Comment by ViktorHofer** [comment]
@ericstj @anipik @joperezr it's that time again:

> src\libraries\pkg\test\testPackages.proj(214,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) Validation Failed: System.Security.Cryptography.Pkcs is harvesting assets from package version 5.0.0 which is not the latest for that package release. Latest package version from that release is 5.0.1. In order to fix this, run `dotnet msbuild System.Security.Cryptography.Pkcs.pkgproj /t:UpdateHarvestVersionOnPackageIndex /p:UpdateStablePackageInfo=true`

**Comment by ViktorHofer** [comment]
Affecting all PRs targeting master

**Comment by ViktorHofer** [comment]
@jkotas we can ignore the libraries allconfigurations error which happens on all PRs right now. Should we merge this in now that we have good coverage? Maybe wait another 30min and then merge?

**Review by trylek — APPROVED** [review]
Thank you!

**Pull request merged**